test/extended/util/annotate/generated/zz_generated.annotations.go

```diff
@@ -1753,8 +1759,6 @@ var annotations = map[string]string{
 
 	"[Top Level] [sig-cli][Feature:LegacyCommandTests][Disruptive][Serial] test-cmd: test/cmd/builds.sh [apigroup:image.openshift.io][apigroup:config.openshift.io]": "test/cmd/builds.sh [apigroup:image.openshift.io][apigroup:config.openshift.io]",
 
-	"[Top Level] [sig-cli][Feature:LegacyCommandTests][Disruptive][Serial] test-cmd: test/cmd/completions.sh [apigroup:image.openshift.io][apigroup:config.openshift.io]": "test/cmd/completions.sh [apigroup:image.openshift.io][apigroup:config.openshift.io]",
```

Comment:
@ingvagabund  completions only tests cli completions and as far as I can see, it does not depend on any apigroup?, I removed these apigroups in new completions tests.

Reply:
Even better :) I made some assumptions about the code when I was going through it which is not always bullet proof.